org.opent2t.sample.hub.superpopular/com.wink.hub/js/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "opent2t-wink-hub",
```

Comment:
We are using a different pattern... see https://github.com/openT2T/translators/issues/50. I believe this should be opent2t-translator-com-wink-hub

We should probably doc this somewhere... I opened #50 to track.